src/actions/Bond.actions.ts

```diff
@@ -1,17 +1,66 @@
-import { ethers } from "ethers";
-import { isBondLP, getMarketPrice, contractForBond, contractForReserve, addressForAsset, bondName } from "../helpers";
-import { addresses, Actions, BONDS } from "../constants";
+import { BigNumber, ethers } from "ethers";
+import {
+  isBondLP,
+  getMarketPrice,
+  contractForBond,
+  contractForReserve,
+  addressForAsset,
+  toNum,
+  bondName,
+} from "../helpers";
+import { addresses, Actions, BONDS, Nested } from "../constants";
 import { abi as BondCalcContract } from "../abi/BondCalcContract.json";
+import { StaticJsonRpcProvider } from "@ethersproject/providers";
+import { IBondData } from "src/reducers";
+import { Dispatch } from "redux";
+import { OlympusBondingCalculator } from "src/typechain";
 import { clearPendingTxn, fetchPendingTxns } from "./PendingTxns.actions";
 
-export const fetchBondSuccess = payload => ({
+interface IChangeApproval {
```

Comment:
What are your thoughts on moving types/interfaces to a separate file/directory? that way the logic isn't cluttered with type definitions.

Reply:
yeah I definitely think we should do this, how should we organize these?

Reply:
another folder under `src` named types or interfaces?

Reply:
Okay I'll try to organize it by category: a file for actions, hooks, etc.